contracts/interchain-token-service/src/interface.rs

```diff
@@ -49,6 +51,14 @@ pub trait InterchainTokenServiceInterface: AxelarExecutableInterface {
         gas_token: Token,
     ) -> Result<BytesN<32>, ContractError>;
 
+    fn deploy_remote_canonical_token(
```

Comment:
add docstring

Reply:
I think we can pick up this task and add all docstring once the code is almost finalized: https://axelarnetwork.atlassian.net/browse/AXE-5852